test/dynamics/signals/test_signals.py

```diff
@@ -728,6 +738,7 @@ def test_conjugate(self):
         )
 
 
+@partial(test_array_backends, array_libraries=["numpy", "jax", "array_numpy", "array_jax"])
```

Comment:
This decorator works as creating TestDiscreteSignalSum of "numpy", "jax", "array_numpy", "array_jax", which inherits TestSignalSum. But there is no class whose name is TestSignalSum because it has already been decorated by test_array_backends.